mailscanner/functions.php

```diff
@@ -372,7 +372,6 @@ function html_start($title, $refresh = 0, $cacheable = true, $report = false)
     echo '</td>' . "\n";
 
     echo '<td align="left" valign="top">' . "\n";
-    printColorCodes();
     echo '  </td>' . "\n";
```

Comment:
This empty td can be removed, beware of the column count on other places of the code